x-pack/solutions/search/plugins/search_connectors/server/task.ts

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { Logger, CoreStart, SavedObjectsClient } from '@kbn/core/server';
+
+import type {
+  ConcreteTaskInstance,
+  TaskManagerStartContract,
+  TaskInstance,
+} from '@kbn/task-manager-plugin/server';
+
+import { LicensingPluginStart } from '@kbn/licensing-plugin/server';
+import type {
+  SearchConnectorsPluginStartDependencies,
+  SearchConnectorsPluginSetupDependencies,
+} from './types';
+import {
+  AgentlessConnectorsInfraService,
+  getConnectorsWithoutPolicies,
+  getPoliciesWithoutConnectors,
+} from './services';
+
+import { SearchConnectorsConfig } from './config';
+
+const AGENTLESS_CONNECTOR_DEPLOYMENTS_SYNC_TASK_ID = 'search:agentless-connectors-manager-task';
+const AGENTLESS_CONNECTOR_DEPLOYMENTS_SYNC_TASK_TYPE = 'search:agentless-connectors-manager';
+
+const SCHEDULE = { interval: '30s' };
```

Comment:
Setting this to the largest value you are willing to live with, will be helpful to Kibana's task throughput :-)

I believe a comment in the PR indicated it could be set to `"1m"` which would cut down the executions by 50% (useful!)

Reply:
I'll change to "1m" indeed, it should not hurt us much, and we can iterate on this number later if it's gonna be too much/too little!